jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/AbstractWebSocketConnection.java

```diff
@@ -521,7 +521,7 @@ public void resume()
     {
         ByteBuffer resume = readState.resume();
         if (resume != null)
-            onFillable(resume);
+            getExecutor().execute(() -> onFillable(resume));
```

Comment:
Uhm, we dispatch now for onFillable?
Always?
This seems to be a mistake.

Reply:
We only dispatch for `onFillable()` when resuming. This is so if we get a control frame in the middle of the message eg `PONG`, we don't call the pong `MessageHandler` from within the same thread that is processing the `MessageHandler<InputStream>`. 

The demand in jetty 10 is similar, there is always a call to `getExecutor().execute(this);` when demanding another frame.